_style/method-invocation.md

```diff
@@ -92,35 +92,28 @@ gray area is short, operator-like methods like `max`, especially if commutative:
     // fairly common
     a max b
 
-Symbolic methods which take more than one parameter (they do exist!)
-may still be invoked using infix notation, delimited by spaces:
+Symbolic methods which take more than one parameter are discouraged.
+When they exist, they may still be invoked using infix notation, delimited by spaces:
 
     foo ** (bar, baz)
 
 Such methods are fairly rare, however, and should normally be avoided during API
-design. For example, the use of the `/:` and `:\` methods should be avoided in
+design. For example, the use of the (now deprecated) `/:` and `:\` methods should be avoided in
 preference to their better-known names, `foldLeft` and `foldRight`.
 
 ### Higher-Order Functions
 
-As noted, methods which take functions as parameters (such as `map` or
-`foreach`) should be invoked using infix notation. It is also *possible* to
-invoke such methods in the following way:
+Invoking higher-order functions may use parens or braces, but in
+either case, use dot notation and omit any space after the method name:
 
-    // wrong!
-    names.map { _.toUpperCase }
+    names.map{ _.toUpperCase }
```

Comment:
If this is supposed to be a "good" example then it should be `names.map(...)`, braces should only be used for multiline expressions.

Reply:
also braces aren't even needed for multi-line expressions when the method takes a lambda in scala 3:
```scala
xs.map(x =>
  val a = x
  a + 1
)
```
because `=>` introduces an indentation region.

Reply:
further adjustment pushed